django_app/redbox_app/redbox_core/views.py

```diff
@@ -153,16 +161,56 @@ def remove_doc_view(request, doc_id: str):
 
 @login_required
 def sessions_view(request, session_id: str = ""):
-    # Hard-coding some chat history for now
-    session_history = [
-        {"session_id": "chat1", "title": "Chat 1", "url": "/sessions/chat1"},
-        {"session_id": "chat2", "title": "Chat 2", "url": "/sessions/chat2"},
-    ]
+    chat_history = models.ChatHistory.objects.all().filter(users=request.user)
+
+    messages = []
+    if session_id:
+        current_chat = models.ChatHistory.objects.get(id=session_id)
+        messages = models.ChatMessage.objects.all().filter(chat_history=current_chat)
```

Comment:
you can chain these two request together like:

```suggestion
        messages = models.ChatMessage.objects.filter(chat_history__id=session_id)
```

Reply:
Good to know - thank you